code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1023,9 +1023,11 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		if(ITEM_SLOT_FEET)
 			if(H.num_legs < 2)
 				return FALSE
-			if(DIGITIGRADE in species_traits)
+//			if(DIGITIGRADE in species_traits)
+			if((DIGITIGRADE in species_traits) == !(I.flags_inv & FULL_DIGITIGRADE)) //fulp edit oh god oh fuck
 				if(!disable_warning)
-					to_chat(H, "<span class='warning'>The footwear around here isn't compatible with your feet!</span>")
+//					to_chat(H, "<span class='warning'>The footwear around here isn't compatible with your feet!</span>")
+					to_chat(H, "<span class='warning'>These shoes aren't compatible with your feet!</span>") //fulp edit 2, electric boogaloo
```

Comment:
Was this edit necessary?

Reply:
It's mostly just so it makes more sense, but I can undo it if needed.

Reply:
adding a change like this will cause possible conflicts in the future and it's just for some slight flavor. Please remove it.